ssl/statem/statem_srvr.c

```diff
@@ -1850,7 +1881,7 @@ static int tls_early_post_process_client_hello(SSL *s, int *pal)
      * Given s->session->ciphers and SSL_get_ciphers, we must pick a cipher
      */
 
-    if (!s->hit || s->hello_retry_request) {
+    if (!s->hit || SSL_IS_TLS13(s)) {
```

Comment:
No, it is relevant. The point is that we are now going to select a ciphersuite in TLSv1.3 whether or not we have resumed. In <= TLSv1.2 we only did this if we did not resume (because a resumption implies we've already selected the ciphersuite).